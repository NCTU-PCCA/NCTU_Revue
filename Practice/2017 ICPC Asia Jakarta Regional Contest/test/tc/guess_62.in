47131575611704 46
>>><<<>><>>><<<>><>><><<>><<><><<><<<><<><><>>
