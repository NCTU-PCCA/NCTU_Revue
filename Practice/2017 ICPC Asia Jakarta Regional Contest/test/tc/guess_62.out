1 2 3 47131575611704 47131575611703 47131575611702 4 5 47131575611701 6 7 8 47131575611700 47131575611699 47131575611698 9 10 47131575611697 11 12 47131575611696 13 47131575611695 47131575611694 14 15 47131575611693 47131575611692 16 47131575611691 17 47131575611690 47131575611689 18 47131575611688 47131575611687 47131575611686 19 47131575611685 47131575611684 20 47131575611683 21 47131575611682 22 23
